File: app/channels/request_channel.rb
class RequestChannel < ApplicationCable::Channel
  def subscribed
    logger.info 'Subscribed to RequestChannel'

    stream_from "request_channel"
  end

  def unsubscribed
    logger.info 'Unsubscribed to RequestChannel'
  end

  def speak(data)
    logger.info "RequestChannel, speak: #{data.inspect}"
  end
end

File: app/javascript/channels/request_channel.js
import consumer from "./consumer"

consumer.subscriptions.create("RequestChannel", {
  connected() {
    // Called when the subscription is ready for use on the server
    console.log('Connected to RequestChannel')
  },

  disconnected() {
    // Called when the subscription has been terminated by the server
    console.log('Disconnected to RequestChannel')
  },

  received(data) {
    // Called when there's incoming data on the websocket for this channel
    $('#messages').prepend(data.message)
    console.log('Recived data: ' + data)
  },

  speak: function() {
    return this.perform('speak', { message: message });
  }
});
